authservice/domain/port/token_repository_interface.go

```diff
@@ -0,0 +1,10 @@
+package port
+
+import "AdsService/authservice/domain/entity"
+
+type TokenRepository interface {
+	GenerateAccessToken(userID uint64, email, role string) (string, error)
+	GenerateRefreshToken(userID uint64) (string, error)
+	ParseAccessToken(tokenStr string) (*entity.AccessClaims, error)
```

Comment:
```suggestion
	ParseAccessToken(token string) (*entity.AccessClaims, error)
```
по типу и так понятно что string